test/extended/util/test.go

```diff
@@ -439,6 +439,8 @@ var (
 			`Job should run a job to completion when tasks sometimes fail and are not locally restarted`, // seems flaky, also may require too many resources
 			`openshift mongodb replication creating from a template`,                                     // flaking on deployment
 			`should use be able to process many pods and reuse local volumes`,                            // https://bugzilla.redhat.com/show_bug.cgi?id=1635893
+
+			`[sig-storage] Volume limits should verify that all nodes have volume limits`, // flaking due to a kubelet issue
```

Comment:
I saw this [again][1] in a job kicked off after the merge.  Maybe the leading `[sig-storage]` here is a problem?  The entries above don't seem to have those.

[1]: https://github.com/openshift/installer/pull/1100#issuecomment-457835937

Reply:
Yeah, it needed to be reflex quoted.  Manually check and merged a follow up.